repository/RediStick-Core.package/RsRedisEndpoint.class/instance/mSet..st

```diff
@@ -1,3 +1,4 @@
 commands-strings
 mSet: aDictionary
-	^ self unifiedCommand: {'MSET'} , (self keysAndValuesInlined: aDictionary)
\ No newline at end of file
+	^ self unifiedCommand: {'MSET'} , (self keysAndValuesInlined: aDictionary)
```

Comment:
MSET tests are also unrelated to the PR topic.

Reply:
done 
7dc7ee4